lib/split/combined_experiments_helper.rb

```diff
@@ -15,9 +16,15 @@ def ab_combined_test(metric_descriptor, control = nil, *alternatives)
             alternative = ab_test(combined_experiment, normalized_alternatives[0], *normalized_alternatives[1])
           end
         else
-          ab_test(combined_experiment, [{alternative => 1}])
+          weighted_alternatives ||= experiment[:alternatives].each_with_object({}) do |alt, memo|
+            alt = Alternative.new(alt, experiment[:name]).name
+            memo[alt] = (alt == alternative ? 1 : 0)
+          end
```

Comment:
If there's a better way to do this, I'm all ears.

The goal is to return an array with a hash that has each alternative with a weight of zero except the selected alternative which should have a weight of 1.

e.g. `[{"control" => 0, "test-alt" => 1}]`

Because `experiment[:alternatives]` can be a simple array or a hash of weights, I'm leaning on `Alternative` to normalize the name.